.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,8 +4,8 @@
 ### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
 <!--- Skip this if not applicable -->
 
-### TEST PLAN
-<!--- What steps should be taken to verify the changes -->
+### TESTING INSTRUCTIONS
```

Comment:
TESTING INSTRUCTIONS(required!)

Reply:
Added this to the comment below the header